app/components/buySell/ChangellyFetcher.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import React, { Component } from 'react';
+import type { Node } from 'react';
+import { action, observable } from 'mobx';
+import { intlShape, } from 'react-intl';
+import type { $npm$ReactIntl$IntlFormat } from 'react-intl';
+import { observer } from 'mobx-react';
+
+type Props = {|
+  +children?: Node,
+  +widgetURL: string,
+  +address: ?string,
+|};
+
+@observer
+export default class ChangellyFetcher extends Component<Props> {
+  static defaultProps: {|children: void|} = {
+    children: undefined
+  };
+
+  @observable iframe: ?HTMLIFrameElement;
+  @observable frameHeight: number = 0;
+
+  @action setFrame: (null | HTMLIFrameElement) => void = (frame) => {
+    this.iframe = frame;
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this.resize);
+    this.resize();
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.resize);
+  }
+
+  static contextTypes: {|intl: $npm$ReactIntl$IntlFormat|} = {
+    intl: intlShape.isRequired,
+  };
+
+  addAddressToWidget: (string, string) => string = (widgetURL, address) => {
+    if (address == null) return widgetURL
+    return widgetURL + '&address=' + address
+  }
+
+  render(): Node {
+    const { widgetURL, address } = this.props;
+
+    if (widgetURL == null) {
+      throw new Error('Changelly URL undefined. this should never happen');
+    }
+
+    return (
+      <iframe
+        /**
+          * Rationale for allowing the following:
+          *
+          * allow-scripts:
+          * The iframe uses javascript for:
+          *   - Integrate Changelly Widget
+          *
+          * allow-popups:
```

Comment:
Do we need `allow-popups`? Looks like the rationale wasn't updated

Reply:
Yeah I tried removing it but for some flows it opens up a new website to continue the flow and as expected it doesn't work without it

Reply:
Okay, maybe just update the comment to mention this then?